client/js/app.modules/main/main.module.js

```diff
@@ -0,0 +1,8 @@
+var angular = require('angular');
+
+var mainModule = angular.module('mainModule', [])
+    .controller('mainCtrl', require('./main.controller.js'))
+    .directive("ngHeaderBlock", require('./ng-header-block.directive.js'))
+    .factory('mainPageInfo', require('./main-page-info.factory.js'));
```

Comment:
I get use to name services in UpperCamelCase (MainPageInfo). But if team decided to use lowerCamelCase its OK, leave it as is.
I'll repeat - the main thing is to  use same convention through the team.